### DISCUSSION TITLE
Plug in window will not resize

### DISCUSSION THREAD

**Discussion by dasmeeth** [apohl79/audiogridder#1103] [Help]
I tried both the release and then the beta versions. The plugin window shows about 1/10 of the plugin and will not resize. Both server and plugin are running on Win 11 intel I7 machines. Does not matter if I use generic controls or not. 
One thing I will add is the display resolutions between the server and plugin PC are different. Server is 1024x768 and plugin is 1920x1080. I will try another monitor  and match resolutions later tonight.

OK, matched resolutions between the 2 machines, no change. Tried 2 different host programs, Wavelab and Audacity. Again no change.

![Gridder1](https://user-images.githubusercontent.com/119460091/204666055-f0396699-99a9-4bf9-9c73-b4cf403a7057.jpg)
![Gridder2](https://user-images.githubusercontent.com/119460091/204666723-c2388ee2-db57-4257-bea9-eac8e8f72591.jpg)

**Comment by apohl79** [comment]
It looks like the host applications do not support resizable plugin UI's? I have not tested any of the two.

**Reply by dasmeeth** [reply]
I will try a few others I have around in the near future.
